ams/ams-server/src/main/java/com/netease/arctic/ams/server/optimize/BaseOptimizeCommit.java

```diff
@@ -258,15 +258,6 @@ public long commit(TableOptimizeRuntime tableOptimizeRuntime) throws Exception {
                 "add {} new files [{} posDelete files]",
             arcticTable.id(), majorDeleteFiles.size(), deleteDeleteFiles.size(), majorAddFiles.size(),
             addDeleteFiles.size());
-
-        // update table runtime base snapshotId, avoid repeat Major plan
```

Comment:
What about the risk of not update base snapshotId in table runtime after commit?

Reply:
no risk.
Because check base table is changed accord to this base snapshotId is equal to current snapshotId before.
Now accord to whether have new append not produced by optimize between  this base snapshotId and current snapshotId